### DISCUSSION TITLE
Lazer Mico-Stutters

### DISCUSSION THREAD

**Discussion by megu-osu** [ppy/osu#26690] [Q&A]
During gameplay, I am consistently getting random stutters that weren't visible to the in-game fps counter or screensharing.
I was suggested by spaceman_atlas to Ctrl+F11 twice which pulled up some graphs.
I realise that the stutters happen at the same time as these thin lines show up for one frame in the graphs.

As requested, here are logs and recording.
It might not be visible in the video, but I feel stutters every time you see the lines pop up.

https://github.com/ppy/osu/assets/78813028/a3c7cc90-6dc7-48cd-8046-1e008165bafb
[compressed-logs.zip](https://github.com/ppy/osu/files/14040470/compressed-logs.zip)

**Comment by bdach** [comment]
Yeah these look like GCs firing so likely no easy fix other than just clamping down on allocations. That said you're running on 1000fps which is probably overkill. I'd recommend dropping down to 2x or 4x and seeing if it's better.

**Reply by megu-osu** [reply]
No, I was already playing on 4x, which is why it was 576hz  at the bottom right.

**Reply by bdach** [reply]
Right okay I forgot that update runs at 2x draw (up to 1000).

I'd probably try dropping to 2x limiter and seeing what happens.

**Reply by megu-osu** [reply]
still stuttering, i don't think it's a frame limiter issue
